src/common/Network/PacketDef/Zone/ClientZoneDef.h

```diff
@@ -40,9 +40,9 @@ struct FFXIVIpcClientTrigger :
   /* 0004 */ uint32_t param11;
   /* 0008 */ uint32_t param12;
   /* 000C */ uint32_t param2;
-  /* 0010 */ uint32_t param4; // todo: really?
-  /* 0014 */ uint32_t param5;
-  /* 0018 */ uint64_t param3;
+  /* 0010 */ uint32_t param3;
+  /* 0014 */ uint32_t param4;
+  /* 0018 */ uint64_t param5;
```

Comment:
99% sure these are used in a specific order elsewhere which aren't in this PR, so renaming them won't do what you intend and will instead break other things